pandas/tests/reshape/test_concat.py

```diff
@@ -2927,3 +2927,20 @@ def test_concat_preserves_extension_int64_dtype():
     result = pd.concat([df_a, df_b], ignore_index=True)
     expected = pd.DataFrame({"a": [-1, None], "b": [None, 1]}, dtype="Int64")
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    ("keys", "integrity"),
+    [
+        (["red"] * 3, False),
+        (["red", "blue", "red"], False),
+        (["red", "blue", "red"], True),
+    ],
+)
```

Comment:
Was there a plan to follow the same exact parametrization as in the referenced github issue?
I would suggest to have these tests with ``verify_integrity=True/False`` for all ``keys``.

Reply:
Was not mentioned as a problem, so I think that was the reason I did not add it. But was more than 4 weeks ago, so I am not quite sure. Added it now.